### PR TITLE
Functional tests for 2.0 API OBM

### PR DESCRIPTION
**Pull request by dalebremner**
Depends on RackHD/on-http#173

### CODE REVIEW

**Comment by dalebremner** [comment]
@RackHD/corecommitters @abildw @srinia6 @geoff-reid

File: test/tests/api/v2_0/obm_settings.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+from on_http_api2_0 import rest
+from modules.logger import Log
+from json import dumps, loads
+
+LOG = Log(__name__)
+
+"""
+Class to abstract the RackHD Out-of-Band settings 
+"""
+class obmSettings(object):
```
Comment (jlongever) [inline test/tests/api/v2_0/obm_settings.py]: If there aren't a lot of data structure differences here, it might make more sense to move obm_settings.py to /modules and pass in the API object to the constructor?

Comment (jlongever) [inline test/tests/api/v2_0/obm_settings.py]: Hrm, API method name differences.  Still might be worth looking into making this module common.

**Comment by dalebremner** [comment]
test this please

**Comment by dalebremner** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by dalebremner** [comment]
test this please

**Comment by brianparry** [comment]
:+1:

File: test/tests/api/v2_0/obm_settings.py (hunk 1 above)
Comment (dalebremner) [inline test/tests/api/v2_0/obm_settings.py]: @jlongever sounds good! is that something we can do on a separate PR?

**Comment by geoff-reid** [comment]
:+1:
